subjects/zappy/README.md

```diff
@@ -162,52 +165,52 @@ The time unit is defined by the function 1/t :
 
 - t is passed as parameter at the start of the server.
 - A time unit lasts 1/t seconds.
-
-- There cannot be any blocking when the clients are stopped, or in any phase of the game.
 - The Minerthians have second as an absolute time unit.
+- The formula for the execution of an action is `action/t`.
 - `t` is an integer. If `t=1` "advance" takes 7 seconds. We choose by default, `t=100` then "advance" will take 1/100 seconds.
 
-### Objects denomination
+### Objects Management
+
+It is impossible to distinguish two objects of the same class,. Only the class of an object can be identifiable, that is, two 'ambers' will have the same denomination, as they belong to the same class, but `food` will have a different class than the `stones`, that way you are able to identify the resources.
 
-It is impossible to distinguish two objects of the same class, only the classification of an object can be identifiable,
-that is, two 'ambers' will have the same denomination, as they belong to the same class.
+### Teams/Families
 
-### Teams
+The purpose is to create a `team/family` of six players and get them to level 8!
```

Comment:
Here you can say `the purpose of the game` or `the goal of the game`. Only `the purpose` sounds a bit off.

Reply:
done